packages/sitecore-jss/src/graphql-request-client.test.ts

```diff
@@ -3,27 +3,38 @@ import nock from 'nock';
 import { GraphQLRequestClient } from './graphql-request-client';
 
 describe('GraphQLRequestClient', () => {
-  const ENDPOINT = 'http://jssnextweb/graphql';
-
-  const graphQLClient = new GraphQLRequestClient(ENDPOINT);
+  afterEach(nock.cleanAll);
 
-  const mockGraphQLRequest = () => {
+  it('should execute graphql request', async () => {
     nock('http://jssnextweb')
       .post('/graphql')
       .reply(200, {
         data: {
           result: 'Hello world...',
         },
       });
-  };
-
-  afterEach(nock.cleanAll);
-
-  it('should execute graphql request', async () => {
-    mockGraphQLRequest();
 
+    const graphQLClient = new GraphQLRequestClient('http://jssnextweb/graphql');
     const data = await graphQLClient.request('test');
 
     expect(data).to.deep.equal({ result: 'Hello world...' });
   });
+
+  it('should send sc_apikey header', async () => {
```

Comment:
Also I see in the graphql api docs that tokens can have different scopes - are there error responses relating to this that we need to test for?

Reply:
@anastasiya29 Which scopes are you referring to? I'm not aware of any for authentication.

Reply:
@anastasiya29 I see now. Those scopes are used when provisioning the API keys, but we shouldn't have to worry about them with our authentication. Currently, we don't have any special handling for failed calls (invalid or missing sc_apikey, unauthorized, etc) - so those would go through our current logging and then trickle up to caller to handle. For now, I think it makes sense to keep it this way until we have a reason to do otherwise.